src/pages/CoursePage.js

```diff
@@ -373,18 +373,20 @@ export default class CoursePage extends React.Component {
 
 const ColorHeader = styled.div`
   width: 100vw;
-  height: calc(14vmin + 5rem);
+  height: calc(14vmin + 8rem);
```

Comment:
Haven't had the chance to check completely but the changes made within this file cause weird layout errors on mobile devices (i.e. color header too tall or too much space around the course description).

Reply:
I see the issue. The color header had a set value which meant that longer course names overflowed, and shorter ones had too much space. I've changed it to be relative to the course name, so there's a fixed padding at the top.